packages/odyssey-react/src/components/Infobox/index.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import type { FunctionComponent, ReactNode } from 'react';
+import { useCx, useOmit } from '../../utils';
+
+export type InfoboxVariants = 'info' | 'danger' | 'caution' | 'success';
+
+export type Props = {
+  /**
+   * Content to be rendered within the infobox. Avoid using direct children, put child content
+   * within the provided Infobox static components (Infobox.Content and Infobox.Actions)
+   */
+  children: ReactNode,
+  
+  /**
+   * The visual variant to be displayed to the user.
+   * @default info
+   */
+  variant?: InfoboxVariants,
+
+  /**
+   * The title or headline of the Infobox. If Infobox.Content is not present it is required.
+   */
```

Comment:
What is the relationship between the `title` prop and `Infobox.Content`? Having one required when the other is not present is something we should enforce through code if it is critical.

Reply:
Let's talk about this in OH today!